paper-input-behavior.html

```diff
@@ -264,10 +264,23 @@
     /**
      * Validates the input element and sets an error style if needed.
      */
-     validate: function () {
+     validate: function() {
        return this.inputElement.validate();
      },
 
+    /**
+     * Restores the cursor to its original position after updating the value.
+     */
+    updateValueAndPreserveCaret: function(newValue) {
+      var start = this.inputElement.selectionStart;
```

Comment:
Oh god, that, and I forgot (again) that `<input type=number>` can't use selectionStart.